src/chart/detailTable/webchartsTable/draw.js

```diff
@@ -0,0 +1,8 @@
+export function draw(chart, data) {
+    chart.detailTable.table = webCharts.createTable(
+        //chart.config.container + ' .aeExplorer .aeTable .table-wrapper .DetailTable',
+        '.DetailTable',
```

Comment:
This needs to be a unique selector, such as `chart.detailTable.wrap.node()` or something.

Reply:
oops - forgot to fix this. We should make that update before releasing. @samussiah you have time to put in a fix?

Reply:
Updated.